integreat_cms/cms/forms/custom_content_model_form.py

```diff
@@ -230,7 +230,16 @@ def add_success_message(self, request: HttpRequest) -> None:
         :param request: The current request submitting the translation form
         """
         model_name = type(self.instance.foreign_object)._meta.verbose_name.title()
-        if not self.instance.status == status.PUBLIC:
+        if model_name == "Page" and not request.user.has_perm(
```

Comment:
`model_name` is "Seite" if German is being selected as the user language and the error remains exits 🙈

Reply:
Whoops, good catch 😄